src/codemap_details/Components/WhyCodeMap.js

```diff
@@ -1,6 +1,42 @@
-import React from "react";
+import React, { Component } from "react";
+import { Jumbotron, Container } from "react-bootstrap";
+import { Card } from "react-bootstrap";
+import box from "../assets/box.svg";
 import styles from "../CSS/WhyCodeMap.module.css";
 
-export default function WhyCodeMap() {
-  return <React.Fragment />;
+export default class WhyCodeMap extends Component {
+  render() {
+    return (
+      <Jumbotron  style={{ backgroundColor: "#ffffff" }}>
```

Comment:
@Meghag5 remove inline styling from everywhere and set bottom padding of jumbotron to be 0px.